src/drivers/fluid_alsa.c

```diff
@@ -1100,7 +1164,7 @@ new_fluid_alsa_seq_driver(fluid_settings_t *settings,
     FLUID_MEMSET(port_info, 0, snd_seq_port_info_sizeof());
 
     fluid_settings_getint(settings, "synth.midi-channels", &midi_channels);
-    dev->port_count = midi_channels / 16;
+    dev->port_count = ceil(midi_channels / 16);
```

Comment:
Yes, I think so, I forgot to change one of the operands to float because the original calculation was doing also an integer division, with the same result as floor(). The intention was to create an additional port for the remaining MIDI channels when the number of channels is not a multiple of 16.